sheet2jpk_vat/jpk_vat.py

```diff
@@ -131,7 +132,6 @@ def Write(fo, nip_number, name, email, begin, end, sells, buys, version=0, sysna
 					for net_value, tax_percent, tax_value in i.GroupByTaxPercents():
 						if tax_percent == '0,00%':
 							xml.tns__K_13(Dec2Str(net_value))
-							xml.tns__K_14(Dec2Str(tax_value))
```

Comment:
Tu chyba był błąd. Pole K_14, to kwota netto (o której mowa w art. 129 ustawy), a nie podatek.